### DISCUSSION TITLE
how can I get the pdf_viewer ref

### DISCUSSION THREAD

**Discussion by NyikaHE** [wojtekmaj/react-pdf#1007] [Q&A] [answered]
I want to do some operate on the pdf_viewer,but I Don't know how to get it

**Comment by NyikaHE** [comment]
I use the "scale" property to scale the page,but the result is like this 
![image](https://user-images.githubusercontent.com/26538113/170675385-f1c7123b-86df-43e7-a96b-f57abee12e94.png)
you can see that I cant't align the window's left with the document's left
what I want is like this
![image](https://user-images.githubusercontent.com/26538113/170676001-5fad101a-3070-439f-80b4-31c23cc08a1c.png)
you can see that while I zoom in that the left is keep aliening
@wojtekmaj

**Comment by wojtekmaj** [comment, marked as answer]
There's no such thing as pdf_viewer ref, I don't know what you mean.

**Reply by NyikaHE** [reply]
sorry，I means that I can't see the left border when I use the property 'scale' to zoom on the document.
![image](https://user-images.githubusercontent.com/26538113/170901296-83240470-39cb-4412-8be2-d716ca89c8f5.png)

**Reply by NyikaHE** [reply]
@wojtekmaj
